lib/routes/account.js

```diff
@@ -1646,11 +1668,46 @@ module.exports = function (
             isVerified = isVerified && sessionVerified
           }
 
-          return {
-            email: sessionToken.email,
-            verified: isVerified,
-            sessionVerified: sessionVerified,
-            emailVerified: emailVerified
+          if (! isVerified) {
+            return bounces.check(sessionToken.email)
```

Comment:
Let's definitely do this 'feature' switch in some form, at the very least it'll give us a quick way to disable the feature if it causes load problems.